micrometer-tracing-tests/micrometer-tracing-integration-test/src/main/java/io/micrometer/tracing/test/reporter/wavefront/WavefrontBraveSetup.java

```diff
@@ -54,6 +39,15 @@
 import io.micrometer.tracing.reporter.wavefront.WavefrontSpanHandler;
 import io.micrometer.tracing.test.reporter.BuildingBlocks;
 
+import java.util.Deque;
```

Comment:
Which import order is the correct?

Reply:
I have no idea. I haven't noticed that I have changed that class